recipes/example/meta.yaml

```diff
@@ -20,6 +20,7 @@ source:
 
 build:
   # Uncomment the following line if the package is pure python and the recipe is exactly the same for all platforms.
+  # It is ok if the dependencies are not built for all platforms/versions, although selectors are still not allowed.
```

Comment:
Nit: maybe replace `ok` with `okay` for the pedants; default-English spelling uses either `okay` or `OK`/`O.K.` ([source](https://english.stackexchange.com/questions/25617/difference-between-ok-and-okay)).

Reply:
Relvent: https://www.youtube.com/watch?v=ytlNgZMpyeY